packages/engine-server/src/engineClient.ts

```diff
@@ -426,7 +426,7 @@ export class DendronEngineClient implements DEngineClient, EngineEventEmitter {
     const existing = await this.api.engineGetNoteByPath({
       vault: note.vault,
       ws: this.ws,
-      npath: note.id,
+      npath: note.fname,
```

Comment:
Would you mind adding a docstring for `npath` on 'GetNoteOptsV2` that it should be a note name? That might help avoid this type of bugs in the future.

Reply:
sure